google/cloud/asset/v1/asset_client.h

```diff
@@ -579,54 +579,8 @@ class AssetServiceClient {
-  ///    "Important".
-  ///  * `displayName:Impor*` to find Google Cloud resources whose display name
-  ///    contains "Impor" as a prefix of any word in the field.
-  ///  * `location:us-west*` to find Google Cloud resources whose location
-  ///    contains both "us" and "west" as prefixes.
-  ///  * `labels:prod` to find Google Cloud resources whose labels contain "prod"
-  ///    as a key or value.
-  ///  * `labels.env:prod` to find Google Cloud resources that have a label "env"
-  ///    and its value is "prod".
-  ///  * `labels.env:*` to find Google Cloud resources that have a label "env".
-  ///  * `kmsKey:key` to find Google Cloud resources encrypted with a
-  ///    customer-managed encryption key whose name contains "key" as a word. This
-  ///    field is deprecated. Please use the `kmsKeys` field to retrieve Cloud KMS
-  ///    key information.
-  ///  * `kmsKeys:key` to find Google Cloud resources encrypted with
-  ///    customer-managed encryption keys whose name contains the word "key".
-  ///  * `relationships:instance-group-1` to find Google Cloud resources that have
-  ///    relationships with "instance-group-1" in the related resource name.
-  ///  * `relationships:INSTANCE_TO_INSTANCEGROUP` to find Compute Engine
-  ///    instances that have relationships of type "INSTANCE_TO_INSTANCEGROUP".
-  ///  * `relationships.INSTANCE_TO_INSTANCEGROUP:instance-group-1` to find
-  ///    Compute Engine instances that have relationships with "instance-group-1"
-  ///    in the Compute Engine instance group resource name, for relationship type
-  ///    "INSTANCE_TO_INSTANCEGROUP".
-  ///  * `state:ACTIVE` to find Google Cloud resources whose state contains
-  ///    "ACTIVE" as a word.
-  ///  * `NOT state:ACTIVE` to find Google Cloud resources whose state doesn't
-  ///    contain "ACTIVE" as a word.
-  ///  * `createTime<1609459200` to find Google Cloud resources that were created
-  ///    before "2021-01-01 00:00:00 UTC". 1609459200 is the epoch timestamp of
-  ///    "2021-01-01 00:00:00 UTC" in seconds.
-  ///  * `updateTime>1609459200` to find Google Cloud resources that were updated
-  ///    after "2021-01-01 00:00:00 UTC". 1609459200 is the epoch timestamp of
-  ///    "2021-01-01 00:00:00 UTC" in seconds.
-  ///  * `Important` to find Google Cloud resources that contain "Important" as a
-  ///    word in any of the searchable fields.
-  ///  * `Impor*` to find Google Cloud resources that contain "Impor" as a prefix
-  ///    of any word in any of the searchable fields.
-  ///  * `Important location:(us-west1 OR global)` to find Google Cloud
-  ///    resources that contain "Important" as a word in any of the searchable
-  ///    fields and are also located in the "us-west1" region or the "global"
-  ///    location.
+  ///     @n
+  ///     For more information, see [SearchAllResourcesRequest][google.cloud.asset.v1.SearchAllResourcesRequest].
```

Comment:
It looks like there is some extra indentation added in these cases.

Reply:
Fixed